packages/next-swc/crates/core/src/next_ssg.rs

```diff
@@ -143,6 +143,21 @@ impl Fold for Analyzer<'_> {
         e
     }
 
+    fn fold_jsx_element(&mut self, jsx: JSXElement) -> JSXElement {
+        match &jsx.opening.name {
+            JSXElementName::Ident(i) => {
+                self.add_ref(i.to_id());
+            }
+            _ => {}
+        }
+
+        jsx.fold_children_with(self)
+    }
+
+    fn fold_export_default_decl(&mut self, export: ExportDefaultDecl) -> ExportDefaultDecl {
```

Comment:
This seems like a redudant implementation as default implementation of `fold_export_default_decl` does the exactly same thing.

Reply:
you're right, this doesn't do anything. removed